drivers/flash/flash_stm32h7x.c

```diff
@@ -60,11 +62,34 @@ static inline void _flash_stm32_sem_give(const struct device *dev)
 #define flash_stm32_sem_give(dev)
 #endif
 
+#define STM32H7_SERIES_MAX_FLASH	2048
+#define BANK2_OFFSET	(KB(STM32H7_SERIES_MAX_FLASH) / 2)
```

Comment:
Should it be defined under `#if DUAL_BANK` ?

Reply:
Done